source/server/server.h

```diff
@@ -357,5 +358,19 @@ class MetricSnapshotImpl : public Stats::MetricSnapshot {
   std::vector<std::reference_wrapper<const Stats::ParentHistogram>> histograms_;
 };
 
+/**
```

Comment:
Those methods should be in `Server` (as static) or `OptionsImpl`

Reply:
I'd prefer to use `Server` so that `OptionsImpl` doesn't diverge from `Options`.

Reply:
I don't see a `Server` class, so `OptionsImpl` it is.

Reply:
Fixed.